Lib/traceback.py

```diff
@@ -481,39 +481,10 @@ def __init__(self, exc_type, exc_value, exc_traceback, *, limit=None,
         # permit backwards compat with the existing API, otherwise we
         # need stub thunk objects just to glue it together.
         # Handle loops in __cause__ or __context__.
+        _is_chained = _seen is not None
```

Comment:
This doesn't need to have a `_` prefix, it's just a local variable.

Reply:
Yes. Before I realized I can use _seen this was another private arg, so had the _.  I'll change to is_recursive_call so it's clear what it's for.